### PR TITLE
Portfolio: mobile version pull request

### PR DESCRIPTION
**Pull request by Ney2**
This is a pull request for the ' Portfolio: mobile version' project.
1. Add the work and about section in index.html file
2. Add styling for work and about section in style.css file

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -48,5 +48,116 @@ <h2>I am a Software Developer</h2>
...
+       <p>Ruby</p>
+     </div>
+     <a href="#">See Project</a>
+   </div>
+   </section>
+
+   <section class="About-me">
+     <h2>About me</h2>
+     <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+    <a href="#">Get My Resume</a>
```
Comment (Chimwemwe-127001) [inline index.html:128]: - [ ] I suggest you follow the Figma as close as possible for this part of the project aswell.

Your design 
Screenshot from 2022-05-28 16-56-21

The figma design
Screenshot from 2022-05-28 16-59-34

**Review by Chimwemwe-127001 — CHANGES_REQUESTED** [review]
# Required Changes ♻️ :warning: 
### Hi @saqibjavaiddev hi
Good job so far! :tada: 

There are some issues that you still need to work on to go to the next project but you are almost there! :pinching_hand: 

so close

## Highlights :+1: 
- Repo looks professional :briefcase:
- README.md looks professional :briefcase:

## Required Changes ♻️
- [ ] You must stick to the design as much as possible (e.g. font, colors, images, tests, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

***Check the comments under the review.***

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Ney2** [comment]
Updated styles.css file to meet the requirements.

File: index.html [hunk 2]
```diff
@@ -48,5 +48,116 @@ <h2>I am a Software Developer</h2>
...
+   <div class="work-pic-2">
+     <h2>Profesional Art<br> Printing Data</h2>
+     <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+     <div class="programming-languages">
+       <p>html</p>
+       <p>Bootstrap</p>
+       <p>Ruby</p>
+     </div>
+     <a href="#">See Project</a>
+   </div>
```
Comment (cwaku) [inline index.html:82]: - [ ] As requested by the previous reviewer, it is required to have your project as identical to the Figma template as possible. The section should use the font and colours (background and font) defined for the template. Kindly consider adding a bit more CSS for us to achieve this and make your project rock 😃

<html>
<body>
<p dir="auto" style="box-sizing: border-box; margin-top: 0px !important; margin-bottom: 16px; color: rgb(201, 209, 217); font-family: -apple-system, BlinkMacSystemFont, &quot;Segoe UI&quot;, Helvetica, Arial, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;; font-size: 14px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(13, 17, 23); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;"></p>

Figma design | Your design
-- | --
image | image
 -- | image





</body>
</html>

File: index.html (hunk 1 above)
Comment (cwaku) [inline index.html:128]: - [ ] I would have made this optional but while you are at it fixing the other changes, kindly change the font here and also make the "Get my resume" button a little bigger as shown in the Figma template.

<html>
<body>
<p dir="auto" style="box-sizing: border-box; margin-top: 0px !important; margin-bottom: 16px; color: rgb(201, 209, 217); font-family: -apple-system, BlinkMacSystemFont, &quot;Segoe UI&quot;, Helvetica, Arial, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;; font-size: 14px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(13, 17, 23); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;"></p>

Your design | Figma design
-- | --
image  |  image



</body>
</html>

**Review by cwaku — CHANGES_REQUESTED** [review]
Hi @Ney2 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Ney2** [comment]
Update styles. css file

File: css/styles.css [hunk 3]
```diff
@@ -108,3 +164,209 @@ body {
...
+
+.title {
+  font-family: 'Crete Round', Arial, sans-serif;
+  font-style: normal;
+  font-weight: 200;
+  font-size: 40px;
+  line-height: 52px;
+  margin-left: 24px;
+  border-bottom: 1px solid rgb(0, 0, 0);
+}
```
Comment (jaflih) [inline css/styles.css:182]: Here the problem is you did't take the good font, you selected the Crete Round italic font, that's why your text is in italic. 
Please go to google font and select the good font
`
@import url("https://fonts.googleapis.com/css2?family=Crete+Round:ital@1&display=swap");
`

File: css/styles.css [hunk 4]
```diff
@@ -108,3 +164,209 @@ body {
...
+  padding: 3px;
+  padding-left: 15px;
+}
+
+.work-pic-1 img {
+  margin-top: 50px;
+  margin-bottom: 10px;
+  margin-left: auto;
+  margin-right: auto;
+  width: 50%;
```
Comment (jaflih) [inline css/styles.css:196]: As asked by the previous reviewer, you have to change the size of your image.
Here you give a value of 50% that's why it doesn't stick to Figma,
Please change 50% by 100% to have it like in the Template 

image

File: css/styles.css [hunk 5]
```diff
@@ -50,6 +69,43 @@ body {
   margin: 0 10px 0 0;
 }
 
+.work-content a {
```
Comment (jaflih) [inline css/styles.css:72]: Please remove the border to stick to Figma

image

File: css/styles.css [hunk 6]
```diff
@@ -108,3 +164,209 @@ body {
...
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  text-align: center;
+  height: 30px;
+  flex: none;
+  padding-left: 16px;
+  justify-content: space-between;
+  border: 1px solid black;
```
Comment (jaflih) [inline css/styles.css:244]: Please remove the border to stick to hte Template
image

File: css/styles.css [hunk 7]
```diff
@@ -108,3 +164,209 @@ body {
...
+  background-repeat: no-repeat;
+  background-position: 100% 25.5%;
+  display: flex;
+  flex-direction: column;
+  margin-top: 30px;
+  padding-top: 30px;
+}
+
+.About-me h2 {
+  font-family: 'Crete Round', Arial, sans-serif;
```
Comment (jaflih) [inline css/styles.css:305]: As required by the previous reviewer, please use the good font

File: css/styles.css [hunk 8]
```diff
@@ -108,3 +164,209 @@ body {
...
+.work-pic-2 {
+  background-image: url("../img/work-img/work-2.png");
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  margin-top: 100px;
+}
+
+.work-pic-2 h2 {
+  font-family: 'Crete Round', Arial, sans-serif;
```
Comment (jaflih) [inline css/styles.css:261]: As required by the previous reviewer, please use the good font

**Review by jaflih — CHANGES_REQUESTED** [review]
Hi @Ney2  

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.




 ## Highlights

Repo looks professional 💼
README.md looks professional 💼

 ## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Ney2** [comment]
Update styles.css file to meet Figma requirement.

**Review by GzimAsani — CHANGES_REQUESTED** [review]
### Hi @Ney2  
 
 Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!
 
 ## Highlights
 * Good job so far :clap: 
 
 ## Required Changes 
- [ ] Number of cards on board should be equal to 15, currently, you have 13. Kindly add those missing
- [ ] Group tasks should be assigned to both of you


 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding! clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 9]
```diff
@@ -48,5 +48,117 @@ <h2>I am a Software Developer</h2>
       </div>
     </section>
 
+    <section class="work-section">
+      <div class="title">
+        <h3>My Recent Works</h3>
+      </div>
+    </section>
+
+ <section class="works">
```
[the remark this answers is not in the document]
Comment (Ney2) [reply index.html:57]: Thank you for your feedback but, I'm not getting what you mean

**Comment by Ney2** [comment]
Can you please tell me what cards I'm missing and also which group task should be assigned ?

**Comment by Ney2** [comment]
-  Number of cards on board should be equal to 15, currently, you have
   13. Kindly add those missing
   -  Group tasks should be assigned to both of you
   - Can you be specific, please?  What are the missing cards? And also
   which group task are you referring to? This is an individual assignment.

**Comment by GzimAsani** [comment]
> Can you please tell me what cards I'm missing and also which group task should be assigned ?

Please ignore that requirement, I have edited. Sorry for the confusion

File: index.html (hunk 9 above)
[the remark this answers is not in the document]
Comment (GzimAsani) [reply index.html:57]: Instead of using display: flex, please use display:grid as it is a requirement

**Comment by Ney2** [comment]
Okay no problem. And thanks for your feedback.

**Comment by Ney2** [comment]
Update index.html file
Update styles.css file

**Review by algerina — APPROVED** [review]
Hi @Ney2 , 

Great work on making the changes requested by a previous reviewer :clap: 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
:heavy_check_mark: Repo looks professional 
:heavy_check_mark: README.md looks professional 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @algerina in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**